<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fatalErrorDialogClass</class>
 <widget class="QWidget" name="fatalErrorDialogClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>448</width>
    <height>180</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>fatalErrorDialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QFrame" name="dialogContainer">
     <property name="styleSheet">
      <string notr="true">border: 0px solid rgb(251,248,240);
background-color: rgb(251,248,240);
border-radius: 5px;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="leftMargin">
       <number>12</number>
      </property>
      <property name="topMargin">
       <number>12</number>
      </property>
      <property name="rightMargin">
       <number>12</number>
      </property>
      <property name="bottomMargin">
       <number>12</number>
      </property>
      <item>
       <widget class="QLabel" name="dialogTitle">
        <property name="styleSheet">
         <string notr="true">font-size: 14px;
color: #66574A;
font-weight: bold;</string>
        </property>
        <property name="text">
         <string>Dialog title</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="dialogContent">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">font-size: 14px;
color: #66574A;</string>
        </property>
        <property name="text">
         <string>Dialog content</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <spacer name="dialogSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>278</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="forceExit">
          <property name="styleSheet">
           <string notr="true">font-size: 14px;
color: #B2732B;
font-weight: bold;</string>
          </property>
          <property name="text">
           <string>关闭程序</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
